client/src/app/__tests__/AppSpec.js

```diff
@@ -2139,6 +2139,22 @@ describe('<App>', function() {
     });
 
 
+    it('should update with attributes getters', async function() {
+
+      // given
+      const tab = await app.createDiagram('form');
+
+      // when
+      const updatedTab = await app.triggerAction('lint-tab', { tab });
+
+      const nameGetter = Object.getOwnPropertyDescriptor(updatedTab, 'name').get;
+      const titleGetter = Object.getOwnPropertyDescriptor(updatedTab, 'name').get;
```

Comment:
This is super low-level. I'd rather expect just a check if the tab name and title are correct (value test).